src/main/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/config/WebClientConfiguration.kt

```diff
@@ -22,32 +15,21 @@ class WebClientConfiguration(
   @Value("\${api.timeout:10s}") val timeout: Duration,
 ) {
   @Bean
-  fun prisonApiWebClient(authorizedClientManager: OAuth2AuthorizedClientManager, builder: WebClient.Builder): WebClient =
+  fun prisonApiWebClient(
+    authorizedClientManager: OAuth2AuthorizedClientManager,
+    builder: WebClient.Builder,
+  ): WebClient =
     builder.authorisedWebClient(authorizedClientManager, registrationId = "prison-api", url = prisonapiRootUri, timeout)
```

Comment:
this mix of calling by name and also by parameter location looks odd - would have expected `timeout` to be by name too